packages/api/endpoints/collections.js

```diff
@@ -145,13 +195,20 @@ async function put({ params: { name, version }, body }, res) {
     );
   }
 
-  const record = await collectionModel.create(body);
+  const dynamoRecord = await collectionsModel.create(collection);
+
+  const dbRecord = dynamoRecordToDbRecord(dynamoRecord);
 
-  if (inTestMode()) {
```

Comment:
This is necessary for the dashboard integration. The reason is that on AWS itself, writing to Dynamo triggers a write to ES using the stream. However, dynamo streams don't exist on Localstack, so we have to manually write the record to ES.

Reply:
I guess I'm having trouble reconciling this comment with this one: https://github.com/nasa/cumulus/pull/1822#discussion_r482085176

Reply:
The difference is that this code here is necessary for being able to use our API locally for testing the dashboard. It's not ideal but required failing some other solution. So we have no choice. Outside of this case though, I'd like to avoid doing `inTestMode()`checks in our production code